src/Microsoft.Deployment.DotNet.Releases/src/SupportPhase.cs

```diff
@@ -8,31 +8,41 @@ namespace Microsoft.Deployment.DotNet.Releases
     /// </summary>
     public enum SupportPhase
     {
-        /// <summary>
-        /// The product is actively supported and will receive updates.
-        /// </summary>
-        Current = 0,
         /// <summary>
         /// The product is considered end-of-life and will not receive any updates.
         /// </summary>
         EOL,
+
         /// <summary>
         /// The product is in long term support and will continue to receive updates.
+        /// See the <see href="https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core">support lifecycle</see>
+        /// document for details on how this value impacts the support duration.
         /// </summary>
         LTS,
```

Comment:
LTS is not a support phase, it is a release type. we should not include this in the enum.

Reply:
Also the releasetype enum should have 2 values - lts and standard (to match the updated releases.json).

Reply:
The releases-index have always used LTS for the support phase, e.g. 6.0 currently looks like `"release-type" : "lts",
            "support-phase": "lts",`
            
Has this changed? 

I thought about adding a new enum, but there was a discussion at one point that if the support phase equals the release type, a product can be considered to be receiving full support. That comparison is not possible if we have two separate types.

Reply:
I'd expect the website to just interpret the existing value and display something else that's more friendly. The library is intended to provide much lower level access I think, but let's see what @mairaw says

Reply:
According to the design doc, we were not doing that breaking change but were adding others to the support-phase.
https://github.com/dotnet/designs/pull/274/files
It would be good to settle on what we want to do here because a lot of people might depend on that field.

support-phase cycle according to that doc:
`preview` -> `go-live` -> [`lts` | `standard`] -> `maintenance` -> `eol`

current cycle is
`preview` -> `rc` -> [`lts` | `current`] -> `maintenance` -> `eol`

For the site, I just check for some specific values and calculate the support phase based on today's date. It would be more straightforward if the support phase would have
`preview` -> `rc` -> `active` -> `maintenance` -> `eol`

/cc @richlander

Reply:
Correct, the library is going to be lower level and something external customers need to be able to rely on for the long term so I don't want that starting off with the incorrect interpretation, it should have the clear separation of release type and support phase without overloading this for website specific needs. we can do something custom/adhoc for the website if we need to, but that shouldn't be in the shipping library. 

Here's how I see these attributes...

release type: lts, standard
support phase: preview -> rc -> active -> maintenance -> eol

Reply:
That sounds good to me @jamshedd. I'll adjust the PR on the designs repo.

Reply:
I see. So this [line](https://github.com/dotnet/designs/pull/274/files#diff-fec46831e913bd6ad143090588ca44b2914e9468f85f2a074d66d85c6dcd0909L213) is what tripped me up. With your suggestion that won't be necessary.

The library already has a fallback check that takes the EOL date into account - we've had instances in the past where products became EOL, but it wasn't reflected in the JSON.

Reply:
Now its perfect :)

Reply:
I'll update the PR once the 7499 PR is updated so I can pull those files into my tests.

Reply:
I thought that we were only going live with the support-phase changes at RTM. So we'd merge 7499 now and the support-phase changes later, no?

Reply:
Now I'm confused. Do I need to start reverting my changes because the library won't work if we make the support phase changes and not update the live data. It's also going to impact some of the internal build processes around servicing.

Reply:
Sounds good @joeloff.